controllers/profiles/followers.js

```diff
@@ -8,7 +8,7 @@ const {QueryTypes} = require('sequelize');
 const config = require('../../databases/config/database')[env];
 
 module.exports = async (req, res) => {
-  const id = req.userId;
+  const userId = req.userId;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The variable name change from `id` to `userId` improves code readability by making it clear that this ID is associated with a user.
```diff
- const id = req.userId;
+ const userId = req.userId;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->